GCRCatalogs/user_config_mgr.py

```diff
@@ -0,0 +1,123 @@
+import os
+import warnings
+import yaml
+from collections.abc import MutableMapping
```

Comment:
Remove unused import (`warnings`).  Import standard modules before external packages.

```suggestion
from collections.abc import MutableMapping

import yaml
```